workspace.html

```diff
@@ -52,40 +52,74 @@ <h3>See the Space</h3>
               {% include modal-form.html initiative='tour' style='img-button' %}
               <!-- Video Modal-->
               <div id="see-space-video-modal" class="modal fade lgx-modal">
-                <div class="modal-dialog">
-                  <div class="modal-content">
-                    <div class="modal-header">
-                      <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    </div>
-                    <div class="modal-body" data-embed="0YgGYadr7rk">
-                      <iframe id="modalvideo" src="https://www.youtube.com/embed/0YgGYadr7rk" allowfullscreen></iframe>
-                    </div>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<style>
+    .bs-example{
+    	margin: 545px;
+    }
+    .modal-content iframe{
+        margin: 0 auto;
+        display: block;
+    }
+</style>
+<script>
+$(document).ready(function(){
```

Comment:
@sgha229 This video doesn't work the same as the internship page for me

Reply:
The issue I had with this page was the format and tools I used for the internship page do not seem to work here for some reason. I'll look into making them all uniform.

Reply:
I've managed to make the video look like the internships page. The issue with the empty YouTube thumbnails that are still appearing at the end of the modal no matter what. Do you have any insight on how to get rid of these pesky things?

Reply:
Never mind, I fixed it. Modal container is a wide size and shape on the page for some reason, poking around the css and js to figure out why.